roles/initialisation/tasks/main.yml

```diff
@@ -3,6 +3,37 @@
     name: bind-utils
     state: latest
 
+- name: Download epel repo rpm
+  get_url:
+    url: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    dest: /tmp/epel-release-latest-7.noarch.rpm
+
+- name: Set permissions on downloaded rpm.
+  file:
+    path: /tmp/epel-release-latest-7.noarch.rpm
+    mode: 0777
```

Comment:
Probably a bit picky but does it need to be executable? Would 0644 not be safer?

Reply:
Changed permission to 0644 and added a task to remove epel rpm once no longer needed.